dwh/validate_indicators.sh

```diff
@@ -64,5 +65,22 @@ if [[ "${counts}" != "34,13,0,4" ]]; then
     "expected to be '34,13,0,4' GOT ${counts}"
   exit 1
 fi
+
+# TX_NEW Test
+counts=$(cat ${TEMP_OUT} | awk -F, '
```

Comment:
After merging your other PR, I think we will have three awk sections like this. Please merge them all into one.